unit-tests/dds/test-options.py

```diff
@@ -30,8 +30,7 @@
     with test.closure( "Test no options" ):
         remote.run( 'test_no_options()' )
         device = dds.device( participant, participant.create_guid(), info )
-        device.run()
-        test.check( device.is_running() )
+        device.wait_until_ready()
```

Comment:
Why did you remove the check if device is ready (as in test-device-init.py)?

Reply:
We test the device init elsewhere, I thought `wait_until_ready()` is pretty clear that it waits until ready.